### ISSUE TITLE
Disable line numbers on parallel columns

### ISSUE THREAD

**Issue by Doc73** [maieul/ledmac#672]
Dear Maieul,
this issue is a follow-up of #368.

As you can remember I would mish regular text with two column text, but in my edition I use inner margins for line numbers and outer margins for marginal note.
If outer column has its line numbers, these numbers may overlap with marginal notes. Therefore, I would suppress line numbers of outer columns.
Is it possible? In `reledpar` doc I can't find a way.

Here is a MWE:
```tex
% !TeX program = xelatex
% !TeX encoding = UTF-8
% !TeX spellcheck = it_IT

\documentclass[b5paper]{book}

\usepackage{fontspec}
\setmainfont{Linux Libertine O}

\usepackage{polyglossia}
\setmainlanguage[babelshorthands=true]{italian}

%%%%% EDIZIONE CRITICA %%%%%
\usepackage{indextools}
\usepackage[noeledsec,noledgroup,series={A,B,C}]{reledmac}
\usepackage[widthliketwocolumns,continuousnumberingwithcolumns]{reledpar}

\labelpstarttrue
\lineation{page}
\linenummargin{inner}
\sidenotemargin{outer}
\firstlinenum*{1}
\linenumincrement*{1}

\AtBeginPairs{\sloppy}
\setlength{\Lcolwidth}{0.47\textwidth}
\setlength{\Rcolwidth}{0.47\textwidth}
\columnsposition{L}
\setlength{\beforecolumnseparator}{0.02\textwidth}
\setlength{\aftercolumnseparator}{0.02\textwidth}
%%%%%%%%%%%%%%%%%%%%

\begin{document}
\beginnumbering
\numberpstarttrue
\pstart
	Paragraph nr. 1] Quel ramo del lago di Como, che volge a mezzogiorno, tra due catene non interrotte di monti, tutto a seni e a golfi, a seconda dello sporgere e del rientrare di quelli, vien, quasi a un tratto, a ristringersi, e a prender corso e figura di fiume, tra un promontorio a destra, e un'ampia costiera dall'altra parte; e il ponte, che ivi congiunge le due rive, par che renda ancor più sensibile all'occhio questa trasformazione, e segni il punto in cui il lago cessa, e l'Adda rincomincia, per ripigliar poi nome di lago dove le rive, allontanandosi di nuovo, lascian l'acqua distendersi e rallentarsi in nuovi golfi e in nuovi seni.
\pend

\pstart \edlabel{itm:Ri33}\edtext{}{\Bfootnote{{\textbf{\pstartref{itm:Ri33}}}\enspace cf. Poll. 5.143, Hesych. σ 2293 (ex Cyr.), sed ex Platonis ipsius verbis quam e fontibus quibusdam sch. sumptum mihi videtur}}333a13 \textit{συμβόλαια Slings: ξυμβόλαια ADT}]\pend

\pausenumbering
\begin{pairs}
\begin{Leftside}
\resumenumbering
\numberpstartfalse
	\pstart\noindent \textbf{a)} σημείωσαι τίνα τὰ ξυμβόλαιά φησιν, ὅτι τὰ κοινωνήματα σημείωσαι τίνα τὰ ξυμβόλαιά φησιν, ὅτι τὰ κοινωνήματα σημείωσαι τίνα τὰ ξυμβόλαιά φησιν, ὅτι τὰ κοινωνήματα σημείωσαι τίνα τὰ ξυμβόλαιά φησιν, ὅτι τὰ κοινωνήματα σημείωσαι τίνα τὰ ξυμβόλαιά φησιν, ὅτι τὰ κοινωνήματα σημείωσαι τίνα τὰ ξυμβόλαιά φησιν, ὅτι τὰ κοινωνήματα. \textbf{A}\pend
\pausenumbering
\end{Leftside}

\begin{Rightside}
\beginnumbering
\numberpstartfalse
	\pstart\noindent\ledsidenote{Bla} \textbf{b)} ὅτι συμβόλαια φησὶ τὰ κοινωνήματα. \textbf{T}\pend
\pausenumbering
\end{Rightside}
\end{pairs}
\Columns

\numberpstarttrue
\resumenumbering

\pstart%
	Paragraph nr. %
\pend

\numberpstarttrue
\resumenumbering

\pstart%
	blablabla%
\pend

\pstart\edlabel{test}%
	\edtext{}{\Bfootnote{{\textbf{\pstartref{test}}}\enspace nel mezzo del cammin}}%
	Paragraph nr. 2%
\pend

\pstart\edlabel{test}%
	\edtext{}{\Bfootnote{{\textbf{\pstartref{test}}}\enspace nel mezzo del cammin}}%
	Paragraph nr. 3%
\pend

\pausenumbering
\begin{pairs}
\begin{Leftside}
\resumenumbering
\numberpstartfalse
	\pstart\noindent Q. Mucius augur multa narrare de C. Laelio socero suo memoriter et iucunde solebat nec dubitare illum in omni sermone appellare sapientem; ego autem a patre ita eram deductus ad Scaevolam sumpta virili toga, ut, quoad possem et liceret, a senis latere numquam discederem; itaque multa ab eo prudenter disputata, multa etiam breviter et commode dicta memoriae mandabam fierique studebam eius prudentia doctior. \textbf{A}\pend
\pausenumbering
\end{Leftside}

\begin{Rightside}
\resumenumbering
\numberpstartfalse
	\pstart\noindent Quinto Mucio l'augure raccontava spesso, a memoria e in modo piacevole, molti episodi della vita di Caio Lelio, suo suocero, e in ogni discorso non esitava a chiamarlo "il Saggio". A Scevola ero stato affidato da mio padre, quando presi la toga virile, perché non mi staccassi mai dal fianco del vecchio, nei \edtext{limiti}{\Cfootnote{llklkj}} del possibile e del consentito. Perciò, fissavo nella mente molti dei suoi accorti ragionamenti e anche molte delle sue massime secche e gustose, e cercavo di migliorare la mia educazione facendo tesoro della sua esperienza di vita. \textbf{B}\pend
\pausenumbering
\end{Rightside}
\end{pairs}
\Columns

\resumenumbering
\numberpstarttrue

\pstart%
	Paragraph nr. 5%
\pend

\pausenumbering
\begin{pairs}
\begin{Leftside}
\resumenumbering
\numberpstartfalse
	\pstart
	Quel ramo del lago di Como, che volge a mezzogiorno, tra due catene non interrotte di monti, tutto a seni e a golfi, a seconda dello sporgere e del rientrare di quelli, vien, quasi a un tratto, a ristringersi, e a prender corso e figura di fiume, tra un promontorio a destra, e un'ampia costiera dall'altra parte; e il ponte, che ivi congiunge le due rive, par che renda ancor più sensibile all'occhio questa trasformazione, e segni il punto in cui il lago cessa, e l'Adda rincomincia, per ripigliar poi nome di lago dove le rive, allontanandosi di nuovo, lascian l'acqua distendersi e rallentarsi in nuovi golfi e in nuovi seni.
	\pend
\pausenumbering
\end{Leftside}

\begin{Rightside}
\resumenumbering
\numberpstartfalse
	\pstart
	Quel ramo del lago di Como, che volge a mezzogiorno, tra due catene non interrotte di monti, tutto a seni e a golfi, a seconda dello sporgere e del rientrare di quelli, vien, quasi a un tratto, a ristringersi, e a prender corso e figura di fiume, tra un promontorio a destra, e un'ampia costiera dall'altra parte; e il ponte, che ivi congiunge le due rive, par che renda ancor più sensibile all'occhio questa trasformazione, e segni il punto in cui il lago \edtext{cessa}{\Cfootnote{llkjlkjj}}, e l'Adda rincomincia, per ripigliar poi nome di lago dove le rive, allontanandosi di nuovo, lascian l'acqua distendersi e rallentarsi in nuovi golfi e in nuovi seni.
	\pend
\pausenumbering
\end{Rightside}
\end{pairs}
\Columns
		
\resumenumbering
\numberpstarttrue
\pstart%
	Paragraph nr. 6%
\pend

\pstart%
	Paragraph nr. 7%
\pend

\pstart%
	Paragraph nr. 8%
\pend

\pstart%
	Paragraph nr. 9%
\pend

\pstart%
	Paragraph nr. 10%
\pend

\pstart%
	Paragraph nr. 11%
\pend

\pstart%
	Paragraph nr. 12%
\pend

\pstart%
	Paragraph nr. 13%
\pend

\pstart%
	Paragraph nr. 14%
\pend

\pstart%
	Paragraph nr. 15%
\pend

\pstart%
	Paragraph nr. 16%
\pend

\pstart%
	Paragraph nr. 17%
\pend

\pstart%
	Paragraph nr. 18%
\pend

\pstart%
	Paragraph nr. 19%
\pend

\pstart%
	Paragraph nr. 20%
\pend

\pstart%
	Paragraph nr. 21%
\pend

\pstart%
	Paragraph nr. 22%
\pend

\pstart%
	Paragraph nr. 23%
\pend

\pstart%
	Paragraph nr. 24%
\pend
\endnumbering
\end{document}
```

In short, I would obtain:
1. Line numbers always in the inner margin (according a book)
2. Marginal notes always in outer margin
3. No line numbers in outer margin

**Comment by maieul** [comment]
No, there is no way, actually.

I need to solve #666 before looking at this problem

**Comment by Doc73** [comment]
Ok, take you time: I have no hurry!

If it could be useful, I would add that mine suggestion is in accordance with ecdotic praxis. I attach, for ex., a page taken from scholia in Lycophronem, by Scheer.
[Uploading scheer2.pdf…]()

**Comment by Doc73** [comment]
Ok, take you time: I have no hurry!

If it could be useful, I would add that mine suggestion is in accordance with ecdotic praxis. I attach, for ex., a page taken from scholia in Lycophronem, by Scheer.

[scheer2.pdf](https://github.com/maieul/ledmac/files/1215382/scheer2.pdf)

**Comment by maieul** [comment]
Since the time I work on reledmac, I can tel you there is no one 
`ecdotic praxis`, but many praxis depending of the domain.

**Comment by Doc73** [comment]
Yes, you're right! 😆

**Comment by maieul** [comment]
Note that you can use `\hidenumbering` in the line you call sidenote. It will hide the line number, but the line number counter will be increased in any case. 

In any case, I will try to get a solution for this issue.

**Comment by maieul** [comment]
I think to this problem tonight, but I will code only tomorrow. I think it should not be difficult, but I have to think to the better way

**Comment by maieul** [comment]
the branch issue673 add two command to do these setting.

They are documented on 1e1c4e7

I have corrected your MWE on c08e945 and  de47a8d.

However, there is still a problem with the side note, as, in column mode, the margin is relative to the column, not to the page.

I will open an issue for that.

**Comment by maieul** [comment]
Hi @doc73 could you look on the new version this branch. It include:
- fix of #674 
- fix of #675 (sidenotesmarginpage option)
- #672 `\linenumOnlyPagesForColumns` and `\linenumOnlyPagesForColumnsR`.

This file https://github.com/maieul/ledmac/blob/c42014e11c3a118cc61e7ecbfd4296364a45eb7a/issue672.tex should be as you want (I have just changed inner/outer margin, but you would obtain what you just changing the setting.)

I will presently add the changelog for the most recent branch for the next release, and then look on your problem.

**Comment by Doc73** [comment]
Oh, thanks!
I have seen the new commands are `\linenumOnlyPagesForColumns` and `\linenumOnlyPagesForColumnsR`, but I can't understand their meaning and their use.
I have made some tests, for ex. with this code
```tex
\setRlineflag{}
\lineation*{page}
\linenumberLevenifblanktrue
\linenumberRevenifblanktrue
\linenummargin{inner}
\linenummarginColumns{right}
\linenummarginColumnsR{left}
\linenumOnlyPagesForColumns{left}
\linenumOnlyPagesForColumnsR{right}
\sidenotemargin*{outer}
```

**Comment by maieul** [comment]
\linenumOnlyPagesForColumns take on argument `left` or `right`.

`\linenumOnlyPagesForColumns{left}` means that for the LEFT column the line number will be printed only on left page.
`\linenumOnlyPagesForColumns{right}` means that for the RIGHT column the line number will be printed only on left page.

Handbook is here https://github.com/maieul/ledmac/commit/1e1c4e7109e3023fc5cff01064482f5010ae8786#diff-2e40dc64935bc56cb699a8c540e2b7a1R1035

**Comment by Doc73** [comment]
Ok, but this solution is very complicated and, if I really undestood it, could lead to pages with line numbers sometimes in inner and sometimes in outer margin.

Is it technically possible to let user to completely disable column numbering and to have only line numbers in inner or outer margins, both in normal text and both in double column text?

**Comment by maieul** [comment]
no, line number are associated with column not with page. But with the combination of all the option, you can get what you want.

**Comment by Doc73** [comment]
Ok, I'll see the doc and search the optimal combination.

**Comment by maieul** [comment]
You want line number inner or outer? i dont remember, but I can do  your MWE.

**Comment by Doc73** [comment]
Only in inner margin, because the outer margin is for marginal notes.

**Comment by maieul** [comment]
so https://github.com/maieul/ledmac/blob/3d75d06e02958c80ed921b1046380fd2d095b665/issue672.tex is for you

**Comment by maieul** [comment]
All my test are ok. I close this issue and  merge branch in release branch. For newest problem, please open a new issue.

**Comment by Doc73** [comment]
Great! Many thanks for your patience!
